lib/chef/resource/zypper_package.rb

```diff
@@ -34,6 +34,10 @@ class ZypperPackage < Chef::Resource::Package
       property :allow_downgrade, [ TrueClass, FalseClass ],
                description: "Allow downgrading a package to satisfy requested version requirements.",
                default: false, introduced: "13.6"
+
+      property :global_options, [ String, Array ],
```

Comment:
@tas50 Please suggest if I need to add `introduced` feild here.
Thanks

Reply:
Yeah, you're going to need to add that.

Reply:
Added, Thank you.